src/languages/ruby.ts

```diff
@@ -167,8 +167,8 @@ const nodeMatchers: Partial<
   ifStatement: "if",
   functionCall: "call",
   comment: "comment",
-  namedFunction: "method",
-  functionName: "method[name]",
+  namedFunction: ["method", "singleton_method"],
+  functionName: ["method[name]", "singleton_method[name]"],
```

Comment:
Please also add to `name` (line 187 below), and add a test for that. Just one test for that one is fine

Reply:
Sorry I missed this suggestion in the issue @richardmcmillen!